CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 2.0.8 - in dev
 
 ### New features
+- luifFriendlyRange : Now shows day of week when range is one day
```

Comment:
reference your issue if it is an issue here

```
 - [#NN](https://github.com/LuccaSA/lucca-ui/issues/NN) - the issue summary 
```

because noone is gonna `git history` the changelog to find this PR and the associated issue